unity-client/Assets/Scripts/MainScripts/DCL/Components/LoadableShapes/NFTShape/Scripts/NFTShapeAssetsType.cs

```diff
@@ -0,0 +1,191 @@
+﻿using System;
+using System.Collections;
+using DCL;
+using DCL.Controllers.Gif;
+using UnityEngine;
+
+namespace NFTShape_Internal
```

Comment:
I dont like this naming, but I have no alternative for this compilation of responsibilities. I'm approving anyway but if time allowes it, give it a think for a couple of minutes in case you can come up with something better!

Reply:
Truth be told, I used that name cause I wanted those classes to be internal but I couldn't 🤣 
Also wanted to create a asmdef for the NFTShape but couldn't either.. a bigger refactor would be needed for that 😢

Reply:
Maybe we could rename this namespace to `NFTShape.Assets`?

Reply:
That will have a naming collision with the NFTShape class